### PR TITLE
assets uglifier for production only? closes #683

### LINKED ISSUE #683: assets uglifier for production only?
hi,

new assets fingerprint and uglifier is awesome 👍 
managed to reduced js from 4mb to 1.6mb.

but there are a concern of performance for development.
the speed to recompile all assets took very long.

is it possible to have webpack command for different environment?
development, staging, production

maybe in webpack.yml (GO_ENV):
```
development: webpack --env.NODE_ENV=$GO_ENV -c ./webpack.config.js -watch
build: webpack --env.NODE_ENV=$GO_ENV -c ./webpack.config.js
```

with this, it will be awesome that we only run build to do uglifier and clean up
what do you think?

here are example:
https://webpack.js.org/guides/environment-variables/

thank you

### PR DESCRIPTION
**Pull request by markbates**

### CODE REVIEW

File: buffalo/cmd/build/assets.go [hunk 1]
```diff
@@ -2,12 +2,14 @@ package build
...
 import (
 	"github.com/gobuffalo/buffalo/generators/assets/webpack"
+	"github.com/gobuffalo/envy"
 	pack "github.com/gobuffalo/packr/builder"
 	"github.com/pkg/errors"
 )
 
 func (b *Builder) buildAssets() error {
 	if b.WithWebpack {
+		envy.Set("NODE_ENV", "production")
```
Comment (u007) [inline buffalo/cmd/build/assets.go]: hi Markbates, 
would be nice if it detects GO_ENV and set it  with default to "production"
and set it into NODE_ENV

File: generators/assets/webpack/templates/webpack.config.js.tmpl [hunk 2]
```diff
@@ -44,48 +104,6 @@ module.exports = {
         screw_ie8: true
       },
       comments: false
-    }),
-    new ManifestPlugin({
```
Comment (u007) [inline generators/assets/webpack/templates/webpack.config.js.tmpl]: manifest is needed for application.[hash].js to work

Comment (markbates) [reply generators/assets/webpack/templates/webpack.config.js.tmpl]: It’s still there. Look at the whole file.

Comment (u007) [reply generators/assets/webpack/templates/webpack.config.js.tmpl]: ohh ya

**Pull request merged**